examples/web/src/features/waku-hub-prototype/browser/styles.css

```
@@ -0,0 +1,198 @@
+:root {
+  --paper: oklch(0.97 0.007 88);
+  --ink: oklch(0.24 0.012 190);
+  --muted: oklch(0.48 0.012 180);
+  --line: oklch(0.84 0.01 88);
+  --accent: oklch(0.43 0.09 215);
+  --accent-soft: oklch(0.93 0.025 215);
+  --focus: oklch(0.55 0.13 215);
+  --ease-out: cubic-bezier(0.23, 1, 0.32, 1);
+  color-scheme: light;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)